android/res/layout/view_rich_notification.xml

```diff
@@ -20,73 +19,71 @@
+            android:text="@string/dummy_title" />
+
+        <TextView
+            android:id="@+id/view_rich_notification_description"
+            style="@style/RichNotificationDescription"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_gravity="left"/>
+            android:layout_gravity="left"
+            android:text="@string/dummy_content_description" />
 
-        <LinearLayout android:id="@+id/view_rich_notification_action_links"
-                      android:layout_width="fill_parent"
-                      android:layout_height="wrap_content"
-                      android:orientation="horizontal"
-                      android:gravity="center_vertical|right">
+        <LinearLayout
+            android:id="@+id/view_rich_notification_action_links"
+            android:layout_width="fill_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center_vertical|right"
+            android:orientation="horizontal">
 
-            <TextView android:text="@string/dummy_extra"
-                      android:layout_width="wrap_content"
-                      android:layout_height="wrap_content"
-                      android:textAppearance="@android:style/TextAppearance.Small"
-                      android:textColor="@color/basic_blue"
-                      android:singleLine="true"
-                      android:textStyle="bold"
-                      android:visibility="gone"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:singleLine="true"
+                android:text="@string/dummy_extra"
+                android:textAppearance="@android:style/TextAppearance.Small"
+                android:textColor="@color/basic_blue"
+                android:textStyle="bold"
+                android:visibility="gone" />
         </LinearLayout>
     </LinearLayout>
-    
-    <View android:id="@+id/view_rich_notification_close_button_vertical_separator"
-        android:layout_width="1sp"
-        android:layout_height="fill_parent"
-        android:layout_marginTop="10sp"
-        android:layout_marginBottom="10sp"
-        android:layout_marginRight="5sp"
-        android:background="@color/app_rich_notification_divider"/>
 
-    <ImageButton android:id="@+id/view_rich_notification_close_button"
+    <TextView
+        android:id="@+id/view_rich_notification_close_button"
+        style="@style/RichNotificationButtonText"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_gravity="center_vertical|center_horizontal"
-        android:background="@null"
-        android:layout_marginRight="5sp"
-        android:src="@drawable/clearable_edittext_clear_icon"/>
-</LinearLayout>
\ No newline at end of file
+        android:text="@string/ok" />
```

Comment:
You  can use `@android:string/ok` instead and delete the new entry in ours `string.xml`